### ISSUE TITLE
Simple LTR implementation for Tweets

### ISSUE THREAD

**Issue by lintool** [lintool/Anserini#31]
@LuchenTan @xeniaqian94 Let's start with a simple two-feature LTR implementation for Tweets:

+ Start with the current tweet search implementation, which has two rerankers, rm3 and cleanup.
+ Your implementation is going to go in a third reranker that you tack on to the end.

Let's build a LTR implementation that just has two features: RM3 score +  number hash tags. Inside your new reranker, you already have the RM3 score; use `getField` on the document to pull out the text, and then just count the number of hashtags. Print out a line like this:

```
1 325263 0.432 3
```
Topic 1, docid 325263, RM3 score of 0.432, 3 hashtags. Dump this information for all docs.

You'll need to take this file and join it with qrels to get the relevance judgments (i.e., write a simple Python script to do it. So you'll end up with a file like:

```
1 325263 0.432 3 1
```

The final column is the relevance judgment. Now you can run learning to rank using http://sourceforge.net/p/lemur/wiki/RankLib/

**Comment by lintool** [comment]
Charlie, take a look at the `ltr` branch for an initial stab at LTR integration. Try:

```
sh target/appassembler/bin/DumpTweetsLtrData -index tweets2011-index/ -num_results 1000 \
 -queries src/main/resources/topics-and-qrels/topics.microblog2011.txt
```

This dumps out the feature vectors for the Tweets 2011 data. We currently have `IntFeatureExtractor` for feature extractors that generate integer based features. If you're happy with this API, we can replicate and add float-based features.

**Comment by lintool** [comment]
Merged in as part of commit 9775cfea8aa20a9e05c1aa8f62e6a8fed40e5817
